sites/partners/src/components/listings/PaperListingForm/RequestChangesModal.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react"
+import { ListingStatus } from "@bloom-housing/backend-core"
+import {
+  AppearanceSizeType,
+  AppearanceStyleType,
+  Button,
+  Form,
+  Modal,
+  t,
+  Textarea,
+} from "@bloom-housing/ui-components"
+import { useForm } from "react-hook-form"
+import { FormListing } from "../../../lib/listings/formTypes"
+
+type FormFields = {
+  requestedChanges: string
+}
+
+type RequestChangesModalProps = {
+  defaultValue: string
+  modalIsOpen: boolean
+  setModalIsOpen: React.Dispatch<React.SetStateAction<boolean>>
+  submitFormWithStatus: (
+    confirm?: boolean,
+    status?: ListingStatus,
+    newData?: Partial<FormListing>
+  ) => void
+}
+
+const RequestChangesModal = ({
+  defaultValue,
+  modalIsOpen,
+  setModalIsOpen,
+  submitFormWithStatus,
+}: RequestChangesModalProps) => {
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, getValues, errors, trigger, clearErrors } = useForm<FormFields>()
+
+  return (
+    <Modal
+      open={modalIsOpen}
+      title={t("t.areYouSure")}
+      ariaDescription={t("listings.approval.requestChangesDescription")}
+      onClose={() => setModalIsOpen(false)}
+      actions={[
+        <Button
+          id="requestChangesButtonConfirm"
+          type="button"
+          styleType={AppearanceStyleType.success}
+          onClick={async () => {
+            const validation = await trigger()
+            if (validation) {
+              const formData = getValues()
+              submitFormWithStatus(false, ListingStatus.changesRequested, {
+                requestedChanges: formData.requestedChanges,
+                requestedChangesDate: new Date(),
+              })
+            }
+          }}
+          size={AppearanceSizeType.small}
+          dataTestId={"requestChangesButton"}
+        >
+          {t("listings.approval.requestChanges")}
+        </Button>,
+        <Button
+          type="button"
+          onClick={() => {
+            setModalIsOpen(false)
+          }}
+          size={AppearanceSizeType.small}
+        >
+          {t("t.cancel")}
+        </Button>,
+      ]}
+    >
+      <Form id={"request-changes"} onSubmit={() => false}>
+        <div className={"pb-4 text-gray-700"}>
+          {t("listings.approval.requestChangesDescription")}
+        </div>
+        <Textarea
```

Comment:
There are a couple of weird quirks with this text box. I don't know if it's because of the TextArea or how it is implemented here:

- I am unable to click enter within the box. Or in other words can't create a new line in the comment.
- This is minor, but I am able to enter html and that is getting formatted on the DetailNotes component. For example `comment comment comment <div>test</div>` will display "test" on a new line. I'm guessing we don't have to be too concerned with that

Reply:
Both of these would be a UIC change, if you think we should update either of these pre-listings-approval launch lmk!

Reply:
I didn't realize that you couldn't put a new line in any Textarea. In that case, I think that that shouldn't be tackled here but we could talk to product/design about if that is a bug on the system as a while and if so create a ticket.

And I'm not too worried about the html issue. With any user inputted field you have the risk of potential security issues if input is not cleaned up. But this is also the current pattern on any Textarea field.

Reply:
Confirmed w product this is good for now and that changes are a part of the larger forms chat